### PR TITLE
Removed codeserver installation from docker image

### PR DESCRIPTION
**Pull request by AhmedSa-mir**
Will be on codeserver package installation instead

See https://github.com/threefoldtech/js-sdk/issues/62

### CODE REVIEW

**Comment by xmonader** [comment]
Why would we do that?

It should be on the image no matter what, if running in the host we can help the user by downloading it, but that should never happen in a container.

Please close if no objection to the above

File: jumpscale/install/codeserver-install.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from jumpscale.core.base import StoredFactory
+from jumpscale.servers.threebot.threebot import ThreebotServer
+
+factory = StoredFactory(ThreebotServer)
+threebot = factory.get("default")
+threebot.packages.add("/sandbox/code/github/js-next/js-sdk/jumpscale/packages/codeserver/")
```
Comment (xmonader) [inline jumpscale/install/codeserver-install.py:6]: that won't work from mac right?

Comment (xmonader) [reply jumpscale/install/codeserver-install.py:6]: you can get the path using `j.packages.codeserver.__path__ ` for codeserver package

Comment (AhmedSa-mir) [reply jumpscale/install/codeserver-install.py:6]: Why?

Comment (abom) [reply jumpscale/install/codeserver-install.py:6]: Also, make sure to save the server.

Comment (AhmedSa-mir) [reply jumpscale/install/codeserver-install.py]: Done

File: jumpscale/install/codeserver-install.py [hunk 2]
```diff
@@ -1,6 +1,8 @@
 from jumpscale.core.base import StoredFactory
 from jumpscale.servers.threebot.threebot import ThreebotServer
+from jumpscale.god import j
 
 factory = StoredFactory(ThreebotServer)
 threebot = factory.get("default")
-threebot.packages.add("/sandbox/code/github/js-next/js-sdk/jumpscale/packages/codeserver/")
+threebot.packages.add(j.packages.codeserver.__file__.rsplit("/", 1)[0])
```
Comment (xmonader) [inline jumpscale/install/codeserver-install.py:7]: why not using j.sal.fs facilities to get accurate paths instead of rsplit, 1, [0] .. .etc?

Comment (AhmedSa-mir) [reply jumpscale/install/codeserver-install.py]: done

**Pull request merged**